dist/configs/config/server.cfg

```diff
@@ -4,9 +4,18 @@
 // 0 - advertise everything, the server will be added to the public server list and information about it will be displayed to players
 // 1 - don't advertise but reply to status queries, this allows players to join and see what map is being played, how many people are playing, etc.
 // 2 - don't reply to status queries but accept connections, players can connect but not see server information
-// 3 - only accept LAN connections
 set server.private 0
 
+// allow source networks for incoming packets
+// 0 - loopback only
+// 1 - LAN
+// 2 - Internet
+set sv_networkScope 2
+
+// contact information to reach out to the server admin when needed: forum or chat nickname, mail address…
+// it is requested to not be empty for public servers
```

Comment:
The wording "to not be empty" seems tortured. How about "to be populated"

Reply:
“to not be empty” is more understandable by non-native english people (very basic words), the “populated” word is very confusing and prone to false-friendliness because its original meaning literally means people (human beings).

Reply:
"to be non-empty"?

Reply:
OK with that.